deepspeed/runtime/swap_tensor/pipelined_optimizer_swapper.py

```diff
@@ -154,6 +154,8 @@ def swap_out_gradients(self, parameter, gradient_offsets, gradient_tensors):
 
     def _complete_swap_out(self, swap_out_type):
         self.swap_ops[swap_out_type].wait()
+        for i in range(len(self.swap_ops[swap_out_type].state_buffers)):
```

Comment:
Minor nit, but you can rewrite this in a more readable form as:
```
for buffer in self.swap_ops[swap_out_type].state_buffers:
    buffer = torch.Tensor()
```

for in range(len()) is a bit of anti-pattern when the index is not strictly needed.

Reply:
@mauryaavinash95, please make this change.

Reply:
Thanks for the suggestion @adk9 and @tjruwase. I've made the change (76b63c888ceaa37e5be0f61149b44b08b3aa486f).